tools/devops/automation/templates/main-stage.yml

```diff
@@ -438,22 +434,4 @@ stages:
       azureContainer: ${{ parameters.azureContainer }}
       packages: # flatten the pkgs for the parameter
       - ${{ each pkg in parameters.legacyPackageJobs }}:
-        - ${{ pkg }}
-
-- ${{ if eq(parameters.enableAPIDiff, true) }}:
-  - stage: generate_api_diff
-    displayName: '${{ parameters.stageDisplayNamePrefix }}API diff'
-    dependsOn: [ configure_build ]
-    jobs:
-    - template: ./build/api-diff-stage.yml
-      parameters:
-        xcodeChannel: ${{ parameters.xcodeChannel }}
-        macOSName: ${{ parameters.macOSName }}
-        isPR: ${{ parameters.isPR }}
-        repositoryAlias: ${{ parameters.repositoryAlias }}
-        commit: ${{ parameters.commit }}
-        vsdropsPrefix: ${{ variables.vsdropsPrefix }}
-        keyringPass: $(pass--lab--mac--builder--keychain)
-        gitHubToken: $(Github.Token)
-        xqaCertPass: $(xqa--certificates--password)
-        pool: ${{ parameters.pool }}
+        - ${{ pkg }}
```

Comment:
👀

Reply:
Removing this means more changes in the template. I prefer to do this step by step. Move the  diff out, move to 1es, clean things.